dusty/commands/test.py

```diff
@@ -70,7 +70,11 @@ def run_app_or_lib_tests(app_or_lib_name, suite_name, test_arguments, should_exi
     expanded_specs = get_expanded_libs_specs()
     spec = expanded_specs.get_app_or_lib(app_or_lib_name)
     test_command = _construct_test_command(spec, suite_name, test_arguments)
-    ensure_test_image(client, app_or_lib_name, expanded_specs, force_recreate=force_recreate)
+    try:
+        ensure_test_image(client, app_or_lib_name, expanded_specs, force_recreate=force_recreate)
+    except ImageCreationError as e:
+        log_to_client('Failed to create test container with error {}'.format(e.code))
+        sys.exit(e.code)
```

Comment:
It would be printed right above 'Failed to create test container with error 127'